[2025-03-28 20:49:25,982][saicinpainting.utils][WARNING] - Setting signal 10 handler <function print_traceback_handler at 0x7f16f479dee0>
[2025-03-28 20:49:25,988][__main__][CRITICAL] - Prediction failed due to [Errno 2] No such file or directory: '../lama/big-lama/config.yaml':
Traceback (most recent call last):
  File "../../lama/bin/predict.py", line 47, in main
    with open(train_config_path, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '../lama/big-lama/config.yaml'

